store/postgres/migrations/2020-03-11-162100_add_data_source_context/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE subgraphs.dynamic_ethereum_contract_data_source ADD COLUMN context text;
```

Comment:
I assume this will initialize the column for all existing dynamic data sources with `null` or an empty string. Will these data sources still load properly then?